alerting/src/main/java/org/hawkular/metrics/alerting/InsertedDataSubscriber.java

```diff
@@ -91,7 +95,6 @@ public void onMetricsServiceReady(@Observes @ServiceReady ServiceReadyEvent even
             } else {
                 events = event.getInsertedData()
                         .filter(m -> m.getType() != MetricType.STRING)
-                        .filter(m -> publish.isPublished(m.getMetricId()))
                         .buffer(publishPeriod, TimeUnit.MILLISECONDS, 100)
```

Comment:
It's not part of the PR. It's a side comment.
But I think that this buffer rate should be parametrized.
It will be very useful on performance tests to evaluate the good threshold for this.